AppImage-package/bundle.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+########################################################################
+# Package the binaries built on Travis-CI as an AppImage
+# By Simon Peter 2016
+# For more information, see http://appimage.org/
+########################################################################
+sudo apt-get -y install coreutils binutils python-tk
+export ARCH=$(arch)
+APP=Scribus
+LOWERAPP=${APP,,}
+GIT_REV=$(git rev-parse --short HEAD)
+echo $GIT_REV
+sudo make install DESTDIR=/$APP/$APP.AppDir
+sudo chown -R $USER /$APP
```

Comment:
Because one day we would want to move to the faster container-based Travis setup where we can’t use sudo (this is currently blocked because using Trusty in Travis requires sudo, but it will be changed in the future).